### PR TITLE
SKCommunications.NateOn version 7.0.6.0

### PR DESCRIPTION
**Pull request by alexkoala**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/19926)

### CODE REVIEW

File: manifests/s/SKCommunications/NateOn/7.0.6.0/SKCommunications.NateOn.locale.ko-KR.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: SKCommunications.NateOn
+PackageVersion: 7.0.6.0
+PackageLocale: ko-KR
+Publisher: SK Communications
+PackageName: NateOn
```
Comment (ItzLevvie) [inline manifests/s/SKCommunications/NateOn/7.0.6.0/SKCommunications.NateOn.locale.ko-KR.yaml:8]:
```suggestion
PackageName: ³×ÀÌÆ®¿Â
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/s/SKCommunications/NateOn/7.0.6.0/SKCommunications.NateOn.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: SKCommunications.NateOn
+PackageVersion: 7.0.6.0
+Installers:
+- Architecture: x86
+  InstallerType: nullsoft
+  InstallerUrl: https://nateondownload.nate.com/download/NATEON70.exe
+  InstallerSha256: 418EF6B5EE6FD98E04D6E13E5D90C378364AD3D7325468CE8613A2B2FA9DE075
```
Comment (ItzLevvie) [inline manifests/s/SKCommunications/NateOn/7.0.6.0/SKCommunications.NateOn.installer.yaml:10]:
```suggestion
  InstallerSha256: 418EF6B5EE6FD98E04D6E13E5D90C378364AD3D7325468CE8613A2B2FA9DE075
  ProductCode: "{EA77EC9A-C82F-4F80-8B7D-D32C09A9C25F}"
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/s/SKCommunications/NateOn/7.0.6.0/SKCommunications.NateOn.locale.ko-KR.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: SKCommunications.NateOn
+PackageVersion: 7.0.6.0
+PackageLocale: ko-KR
+Publisher: SK Communications
+PackageName: NateOn
+License: Copyright (C) SK Communications. All rights reserved.
+ShortDescription: NateOn Setup
```
Comment (ItzLevvie) [inline manifests/s/SKCommunications/NateOn/7.0.6.0/SKCommunications.NateOn.locale.ko-KR.yaml:10]:
```
ShortDescription: NateOn Setup
```
Write a better description for the application.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/s/SKCommunications/NateOn/7.0.6.0/SKCommunications.NateOn.locale.ko-KR.yaml (hunk 1 above)
Comment (alexkoala) [reply manifests/s/SKCommunications/NateOn/7.0.6.0/SKCommunications.NateOn.locale.ko-KR.yaml:8]: image
@ItzLevvie

Comment (ItzLevvie) [reply manifests/s/SKCommunications/NateOn/7.0.6.0/SKCommunications.NateOn.locale.ko-KR.yaml:10]: That's what it shows up as in Control Panel for me; and Windows Package Manager requires the name to match whatever's shown in Control Panel for the best upgrsde experience.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Check the latest review.

**Comment by alexkoala** [comment]
fixed

File: manifests/s/SKCommunications/NateOn/7.0.6.0/SKCommunications.NateOn.locale.ko-KR.yaml (hunk 1 above)
Comment (alexkoala) [reply manifests/s/SKCommunications/NateOn/7.0.6.0/SKCommunications.NateOn.locale.ko-KR.yaml]: fixed

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**